tests/conftest.py

```diff
@@ -126,47 +126,17 @@ def fixture(tmp_dir_session):
 @pytest.fixture(scope='session')
 def pep518_wheelhouse(tmpdir_factory):
     wheelhouse = tmpdir_factory.mktemp('wheelhouse')
-    dist = tmpdir_factory.mktemp('dist')
-    subprocess.run(
-        [sys.executable, '-m', 'build', '--wheel', '--outdir', str(dist)],
-        cwd=str(package_dir.parent.parent),
-        check=True,
-    )
-    (wheel_path,) = dist.visit('*.whl')
-    subprocess.run(
-        [
-            sys.executable,
-            '-m',
-            'pip',
-            'download',
-            '-q',
-            '-d',
-            str(wheelhouse),
-            str(wheel_path),
-        ],
-        check=True,
-    )
-    subprocess.run(
-        [
-            sys.executable,
-            '-m',
-            'pip',
-            'download',
-            '-q',
-            '-d',
-            str(wheelhouse),
-            'build',
-            'colorama',
-            'meson',
-            'ninja',
-            'patchelf',
-            'pyproject-metadata',
-            'tomli',
-            'typing-extensions',
-            'wheel',
-        ],
-        check=True,
-    )
+    meson_python = str(package_dir.parent.parent)
+    # Populate wheelhouse with wheel for the following packages and
+    # their dependencies.  Wheels are downloaded from PyPI or built
+    # from the source distribution as needed.  Sources or wheels in
```

Comment:
Ahh, it's building ninja from source on Cygwin here, that's why it's passing below.